nemo/collections/nlp/modules/common/huggingface/huggingface_utils.py

```diff
@@ -15,21 +15,36 @@
 import os
 from typing import List, Optional
 
-from transformers import (
-    ALBERT_PRETRAINED_MODEL_ARCHIVE_LIST,
-    BERT_PRETRAINED_MODEL_ARCHIVE_LIST,
-    CAMEMBERT_PRETRAINED_MODEL_ARCHIVE_LIST,
-    DISTILBERT_PRETRAINED_MODEL_ARCHIVE_LIST,
-    GPT2_PRETRAINED_MODEL_ARCHIVE_LIST,
-    ROBERTA_PRETRAINED_MODEL_ARCHIVE_LIST,
-    AlbertConfig,
-    AutoModel,
-    BertConfig,
-    CamembertConfig,
-    DistilBertConfig,
-    GPT2Config,
-    RobertaConfig,
-)
+from nemo.collections.nlp.modules.common.megatron.utils import ApexGuardDefaults
+
+try:
+    from transformers import (
+        ALBERT_PRETRAINED_MODEL_ARCHIVE_LIST,
+        BERT_PRETRAINED_MODEL_ARCHIVE_LIST,
+        CAMEMBERT_PRETRAINED_MODEL_ARCHIVE_LIST,
+        DISTILBERT_PRETRAINED_MODEL_ARCHIVE_LIST,
+        GPT2_PRETRAINED_MODEL_ARCHIVE_LIST,
+        ROBERTA_PRETRAINED_MODEL_ARCHIVE_LIST,
+        AlbertConfig,
+        AutoModel,
+        BertConfig,
+        CamembertConfig,
+        DistilBertConfig,
+        GPT2Config,
+        RobertaConfig,
```

Comment:
these configs still exist in the latest transformers. maybe just need to pull these out and import these out side of the try except block?